packages/theme/src/theme/_navs.scss

```diff
@@ -73,41 +73,42 @@
 	}
 
 	&-tabs {
+		background-color: tint($brand-primary, 90);
```

Comment:
I think it should be a separate PR . We need to search occurrences and create a variable if it really needed. And not only for this color but for other ones